docs/framework/configure-apps/file-schema/windows-identity-foundation/claimsauthorizationmanager.md

```diff
@@ -64,7 +64,7 @@ Registers a claims authorization manager for the incoming claims.
  The following XML shows the configuration for a claims authorization manager that implements policy composed of resource-action pairs each of which specifies boolean combinations of the claims that a requestor must possess to perform the action on the resource. The code that implements the claims authorization manager capable of using this policy can be found in the `ClaimsBasedAuthorization` sample.  
   
 ```  
```

Comment:
I wouldn't on this pass, because the script only hit the blocks that had a problem with \\<. I'll open a new issue to address the MIA language ids.